protocol/localstatequery/queries.go

```diff
@@ -654,34 +653,29 @@ type GenesisConfigResultProtocolParameters struct {
 	MinPoolCost           int
 }
 
-// TODO (#864)
-type DebugNewEpochStateResult any
+type DebugNewEpochStateResult cbor.RawMessage
 
-// TODO (#865)
-type DebugChainDepStateResult any
+type DebugChainDepStateResult cbor.RawMessage
 
-// TODO (#866)
-/*
-result	[ *Element ]	Expanded in order on the next rows.
-Element	CDDL	Comment
-epochLength
-poolMints	{ *poolid => block-count }
-maxLovelaceSupply
-NA
-NA
-NA
-?circulatingsupply?
-total-blocks
-?decentralization?	[num den]
-?available block entries
-success-rate	[num den]
-NA
-NA		??treasuryCut
-activeStakeGo
-nil
-nil
-*/
-type RewardProvenanceResult any
+type RewardProvenanceResult struct {
+	_                     struct{} `cbor:",toarray"`
+	EpochLength           int
+	PoolMints             map[ledger.PoolId]int
+	MaxLovelaceSupply     int64
+	DeltaR1               int64
+	DeltaR2               int64
+	R                     int64
+	TotalStake            int64
+	TotalBlocks           int
+	Decentralization      any
+	AvailableBlockEntries int
+	SuccessRate           any
+	RewardPot             int64
+	DeltaT1               int64
+	ActiveStake           int64
+	Pools                 any
+	Desirabilities        any
```

Comment:
We want to avoid `any` here. The purpose of these issues is to fully realize the actual shapes so they can be fully decoded.